app/main/views/organizations.py

```diff
@@ -62,6 +68,39 @@ def add_organization():
     return render_template("views/organizations/add-organization.html", form=form)
 
 
+def get_service_counts_by_status(services):
+    def get_status(service):
+        if not service.get("active"):
+            return ServiceStatus.SUSPENDED
+        elif service.get("restricted"):
+            return ServiceStatus.TRIAL
+        else:
+            return ServiceStatus.LIVE
```

Comment:
I like what you've done here and the use of the enums!  However, I wonder if this method should either be moved to the [`Service` model](https://github.com/GSA/notifications-admin/blob/main/app/models/service.py#L21) itself or if there are existing methods on the model to use instead.

At a quick glance, it looks like there's a few property methods and such on the `Service` model that could probably be consolidated more cleanly to accurately get a current service's status.

Does that make sense to shift this over and/or make use of what's on the model itself and leverage that instead in the statement below that iterates over the services and gets their statuses?